web/js/data/results.js

```diff
@@ -337,29 +351,29 @@ wv.data.results.dividePolygon = function () {
     var ring = granule.geometry['EPSG:4326'].getLinearRing(0);
     var coords = ring.getCoordinates();
     var latlons = [];
-    _.each(coords, function (coord) {
-      var latlon = new L.LatLng(coord[1], coord[0]);
+    loEach(coords, function (coord) {
+      var latlon = new dataHelper.LatLng(coord[1], coord[0]);
       latlons.push(latlon);
     });
-    var result = L.sphericalPolygon.dividePolygon(latlons);
+    var result = dataHelper.sphericalPolygon.dividePolygon(latlons);
     var newPolys = result.interiors;
     var resultMultiPoly = [];
-    _.each(newPolys, function (newPoly) {
+    loEach(newPolys, function (newPoly) {
       var resultPoly = [];
-      _.each(newPoly, function (newCoord) {
+      loEach(newPoly, function (newCoord) {
         resultPoly.push([newCoord.lng, newCoord.lat]);
       });
       resultMultiPoly.push(resultPoly);
     });
     granule.geometry['EPSG:4326'] =
-      new ol.geom.MultiPolygon([resultMultiPoly]);
+      new OlGeomMultiPolygon([resultMultiPoly]);
     return granule;
   };
 
   return self;
 };
 
-wv.data.results.extentFilter = function (projection, extent) {
+export function dataResultsExtentFilter(projection, extent) {
   var self = {};
```

Comment:
When a non-geographic projection is selected, it looks like this function gets called with `extent` being undefined.